theme/templates/base.html

```diff
@@ -60,11 +60,19 @@ <h1><a href="/">{{ SITENAME }}</a></h1>
   </aside>
 
   <main>
+    <div id="funders">
+      <img id="cbbs" src="{{ SITEURL }}/theme/img/cbbs_logo.png" alt="Logo of CBBS">
+      <img id="efre" src="{{ SITEURL }}/theme/img/EFRE_rgb_web.jpg"
+           alt="Signets of Saxony-Anhalt and the European Regional Development Fund">
+    </div>
     <div id='content'>
     {% block content %}{% endblock %}
     </div>
 
     <footer>
+      <p><small>
+          Acknowledgment: funded by the federal state of Saxony-Anhalt and the "European Regional Development Fund" (ERDF 2014-2020), Vorhaben: Center for Behavioral Brain Sciences (CBBS), FKZ: ZS/2016/04/78113
```

Comment:
I'd remove the word "Acknowledgment"

Reply:
Well, it's the exact wording recommended by the CBBS. So, I'd keep it for now.

Reply:
Haha. In that case, it makes sense.

I read it as: "WHAT FOLLOWS IS AN ACKNOWLEDGMENT:" You know, in case the reader misses the point. ;-)

Reply:
I absolutely get your point. I'm just not sure, what's the formal requirement for a statement like this.

Reply:
If it's their language, I think you did the right thing. I say keep it as it is.